cubic-server/main.cpp

```diff
@@ -9,18 +9,88 @@
 #include "interface/InterfaceContainer.hpp"
 #include "logging/Logger.hpp"
 
-argparse::ArgumentParser argParser(int argc, char **argv)
+auto initArgs(int argc, const char *const argv[])
 {
-    argparse::ArgumentParser program("cubic_server");
+    auto program = configuration::ConfigHandler("CubicServer", "0.1.0");
```

Comment:
We should get the version of the server from a define automatically set by the CMake

Reply:
true